test/Dapr.E2E.Test.Actors.Generators/Clients/GeneratedClientTests.cs

```diff
@@ -0,0 +1,107 @@
+// ------------------------------------------------------------------------
+// Copyright 2023 The Dapr Authors
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//     http://www.apache.org/licenses/LICENSE-2.0
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// ------------------------------------------------------------------------
+
+using Dapr.Actors;
+using Dapr.Actors.Client;
+using Xunit.Abstractions;
+
+namespace Dapr.E2E.Test.Actors.Generators.Clients;
+
+public class GeneratedClientTests

```

Comment:
It was a for a couple of reasons:

 - I needed (wanted?) the E2E test to go through multiple sidecars as would be done for in a "real" multi-service application (rather than both sides using the same sidecar, which seemed to be the case when using the existing E2E fixtures).
  - I was nervous about trying to change the existing E2E test fixtures to allow that behavior (as a first "real" E2E test addition).
  - I also preferred the ability to have the test code and service under test code within the same test project, being  reminiscent of other ASP.NET testing patterns and also enabling easier debugging of both sides.

I think some of the types (e.g. `DaprSidecarFactory`) could eventually be pulled out into a common E2E library and reused as the basis of the existing E2E test fixtures.